src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -18,7 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" />
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" />
```

Comment:
You might need to suppress NU5104. https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104. 

Given that we're depending on a preview version, the packaging for RTM (which happens every build) is likely to fail.

Reply:
Added, thanks.